client/src/app/site/pages/meetings/modules/projector/modules/slides/components/poll-slide/components/poll-slide.component.scss

```diff
@@ -125,11 +125,16 @@
         }
 
         h3.structure-level {
-            margin-top: 2px;
+            margin-top: 9px;
             margin-bottom: 0 !important;
             line-height: 28px;
-            font-size: 25px;
-            font-weight: normal;
+            font-size: 18px;
+            font-weight: 500;
```

Comment:
It is important that the height of the `h3` is 30px. Otherwise the columns won't work properly.